### PR TITLE
fix: stop autocannon instance when request is aborted

### LINKED ISSUE #57: Cancel request should be sent to server
Cancel for a request should be sent to the server and FE should wait for it.

At the moment if you start a request with a lot of connection and cancel it, only FE will abort and it is impossible to start a new request until the server has finished, and FE doesn't know when is finished.

### PR DESCRIPTION
**Pull request by ramonmulia**
Closes #57

### CODE REVIEW

**Review by sameer-coder — APPROVED** [review]
LGTM

**Comment by simoneb** [comment]
@sameer-coder just to confirm have you tried this out or just reviewed the code?

**Review by sameer-coder — CHANGES_REQUESTED** [review]
I just reviewed the code earlier. My bad. I should have tested it as well. I tried it now and it works fine with node v14 but does not work with node v16. I had a call with Ramon and he is looking into it.

**Review by jhonrocha — APPROVED** [review]
Confirmed it is working on Node v16.13.2. LGTM

File: packages/autocannon-ui-backend/routes/execute.js [hunk 1]
```diff
@@ -18,7 +18,9 @@ const schema = {
 }
 
 module.exports = async function (fastify) {
+  fastify.register(require('fastify-cors'), { origin: '*' })
   fastify.post('/api/execute', { schema }, function (request, reply) {
+    reply.raw.setHeader('Access-Control-Allow-Origin', '*')
```
Comment (simoneb) [inline packages/autocannon-ui-backend/routes/execute.js:23]: Would this not work with fastify-cors alone? If so, why?

Comment (ramonmulia) [reply packages/autocannon-ui-backend/routes/execute.js:23]: It doesn't, after trying some approaches for some reason we need to specify `Access-Control-Allow-Origin` on POST to make it work without CORS issue.
Also I opened a question regarding it https://github.com/fastify/help/issues/619

Comment (simoneb) [reply packages/autocannon-ui-backend/routes/execute.js:23]: Understood!

**Review by simoneb — APPROVED** [review]

**Review by sameer-coder — APPROVED** [review]

**Pull request merged**